flow-ethereum-event-watch/src/main/kotlin/jp/co/layerx/cordage/flowethereumeventwatch/flow/EventWatchFlow.kt

```diff
@@ -27,14 +27,17 @@ import java.math.BigInteger
 @InitiatingFlow
 @SchedulableFlow
 class EventWatchFlow(private val stateRef: StateRef) : FlowLogic<String>() {
+
+    lateinit var web3: Web3j
```

Comment:
I think you may define these as values in the `call` function like ones in StartEventWatchFlow.